.github/workflows/composite/maximize-build-space/action.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: maximize-build-space
+description: Maximize build space on github runner
```

Comment:
Should the name `Github` be capitalized?

Reply:
Done